apps/web/src/client/websockets.js

```diff
@@ -24,6 +24,71 @@ if (isClient) {
   io.sails.environment = environment
   io.sails.reconnection = true
   socket = io.socket
+
+  // Heartbeat + self-heal reconnect
+  const startSocketHeartbeat = (() => {
+    let started = false
+    return () => {
+      if (started) return
+      started = true
+
+      console.log('startSocketHeartbeat sockets')
```

Comment:
do you wanna leave these in for prod? or make dev only?

Reply:
im going to leave in for now, but remove eventually